### ISSUE TITLE
[Screenshots] Old Screenshots of Meshery Playground needs to be updated

### ISSUE THREAD

**Issue by saurabh100ni** [layer5io/layer5#5342]
### Current Behavior
Currently, [layer5.io/cloud-native-management/meshery](https://layer5.io/cloud-native-management/meshery) has old screenshots of the [Meshery Playground](https://playground.meshery.io/).
### Desired Behavior
- The screenshots/GIFs which contain Meshery Playground need to be updated on the page with the latest version.

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by Raj100679** [comment]
@saurabh100ni Can I work on this?

**Comment by saurabh100ni** [comment]
Go on @SrijanShovit 😎🚀

**Comment by SrijanShovit** [comment]
Hi @saurabh100ni I have signed up for access to Meshery playground

**Comment by saurabh100ni** [comment]
You are good to go @SrijanShovit

**Comment by SrijanShovit** [comment]
Hi @saurabh100ni I am not able to understand which figures to put for screenshots. Do I need to put any screenshots from meshery playground or something specific?

**Comment by saurabh100ni** [comment]
@SrijanShovit ,

When you'll Navigate to the page mentioned in issue description, you will see some screenshots and GIFs with their respective content. We need to update Screenshots and GIFs with same content but with latest version of the Playground.

**Comment by pranay911** [comment]
Can you please assign this issue to me?

**Comment by SrijanShovit** [comment]
Hi @pranay911 I am still working. You may ask for some other issue.

**Comment by SrijanShovit** [comment]
> @SrijanShovit ,
> 
> When you'll Navigate to the page mentioned in issue description, you will see some screenshots and GIFs with their respective content. We need to update Screenshots and GIFs with same content but with latest version of the Playground.

Yeah I can see images and gifs; but can't figure out how to produce those exact reports from meshery playground

**Comment by SrijanShovit** [comment]
There are screenshots of View & Compare Results, Missing prefix in service, Performance Test @saurabh100ni

**Comment by Manishch27** [comment]
Hey @saurabh100ni i would like to work on this issue, can you please assign it to also me?

**Comment by saurabh100ni** [comment]
Go ahead @Manishch27.

**Comment by Manishch27** [comment]
@saurabh100ni I'm trying to setup the project locally according to the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#set-up-your-local-development-environment) all things are going well, but when I run the "make site" command the build gets stuck at Running gatsby-plugin-sharp.IMAGE_PROCESSING at 3-4%. Can anyone help me to solve this problem?

**Comment by saurabh100ni** [comment]
@Manishch27 try using WSL2 if you aren't already using it (it takes long at first tho)

**Comment by Naresh-kumar-Thodupunoori** [comment]
@Manishch27 are you still working on it?? If not then I would like to work on it @saurabh100ni..As I see no update from 2 weeks

**Comment by narasimha-1511** [comment]
hey are you workin gon this issue @Manishch27 ?

**Comment by hargunkaur286** [comment]
@saurabh100ni can I work on this issue?

**Comment by narasimha-1511** [comment]
Go on @hargunkaur286 🚀

**Comment by Ashparshp** [comment]
Any updates @hargunkaur286?

**Comment by STSonyThomas** [comment]
Hey if this issue is still up could I work on it? @Ashparshp @narasimha-1511

**Comment by narasimha-1511** [comment]
As there is no update from @hargunkaur286 
assigning this to @STSonyThomas

**Comment by STSonyThomas** [comment]
@narasimha-1511 so my analysis is that I have to change the following sections
![Screenshot 2024-07-02 095215](https://github.com/layer5io/layer5/assets/75160254/8eadb71f-16f9-49c3-b922-9899f060712e)

The three video snippets in the above section
AND
![image](https://github.com/layer5io/layer5/assets/75160254/305b7114-f8dc-4d95-a331-b4f6abe733e8)
The picture present at the above section right?

**Comment by narasimha-1511** [comment]
yes you need to change these @STSonyThomas

**Comment by Arnavya** [comment]
i would like to work on this issue @narasimha-1511 ?
as there is no update from @STSonyThomas  from a month!!

**Comment by narasimha-1511** [comment]
Go on @Arnavya 🚀

**Comment by souvikinator** [comment]
Is this issue valid? The landing page has changed significantly

**Comment by jagpreetrahi** [comment]
Hey @Rajesh-Nagarajan-11 , could you please assign this issue to me?. I would like to work on this.

**Comment by RajGajjar-01** [comment]
Can you please assign this issue to me ?

**Comment by saurabhraghuvanshii** [comment]
Thanks @RajGajjar-01  hoping Pr soon.

**Comment by RajGajjar-01** [comment]
Hey @saurabhraghuvanshii I have created a PR please review I have been able to add three latest images and will add further images and gif in follow up PRs.

One issue is that I am a beginner in cloud and also opensource so I have not used kanvas that much so images in which stats are being shown have not been pushed yet.

Thank you
